Sources/Comment.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  Comment.swift
+//  SwiftStack
+//
+//  Created by FelixSFD on 07.12.16.
+//
+//
+
+import Foundation
+
+/**
+ Represents a comment to a `Post`.
+ 
+ - author: FelixSFD
+ 
+ - seealso: [StackExchange API](https://api.stackexchange.com/docs/types/comment)
+ */
+public class Comment: JsonConvertible {
```

Comment:
`Post` has properties like `downvoted` which are not part of `Comment`.

Do you have an idea how to call a superclass of `Post` and `Comment`?